task-launcher/src/tasks/shared/trials/afcStimulus.ts

```diff
@@ -1,5 +1,5 @@
 // For all tasks except: H&F, Memory Game, Same Different Selection
-import jsPsychAudioMultiResponse from '@jspsych-contrib/plugin-audio-multi-response';
+import jsPsychHtmlMultiResponse from '@jspsych-contrib/plugin-html-multi-response';
```

Comment:
This looks good. I think we can just replace `@jspsych-contrib/plugin-audio-multi-response` everywhere including SDS and other non afc tasks. I assume the issue will eventually happen in other tasks as well. Before we merge we should deploy this to qa and test it.